app/views/schemes/locations.html.erb

```diff
@@ -0,0 +1,41 @@
+<% title = @scheme.service_name %>
+<% content_for :title, title %>
+
+<%= govuk_back_link(href: request.referer.to_s) %>
+
+<%= render partial: "organisations/headings", locals: { main: @scheme.service_name, sub: nil } %>
+
+<%= render SubNavigationComponent.new(items: scheme_items(request.path, @scheme.id, @scheme.locations.count.eql?(1) ? "1 location" : "#{@scheme.locations.count} locations")) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-three-quarters">
+    <% @locations.each do |location| %>
+      <section class="x-govuk-summary-card govuk-!-margin-bottom-6">
```

Comment:
Should this be using a govuk component? @paulrobertlloyd ?

Reply:
It’s not a GOV.UK component (yet!) but in the interim, we should add our own component, as we’re using this pattern in 2 places now (and there’s a ticket to use it in a third).

Reply:
Not sure where is the other place, but will be happy to extract into separate component in the iteration? @baarkerlounger @paulrobertlloyd

Reply:
Currently used on the review log page (i.e. https://dluhc-core-staging.london.cloudapps.digital/logs/13828/review, https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/blob/06adfbaf6d692a282facee6726a76d3d91d245d3/app/views/form/review.html.erb), and [CLDC-1310](https://digital.dclg.gov.uk/jira/browse/CLDC-1310) is for using summary cards for listing tenants.

Component should take all the properties that can be passed to the summary list component in addition to a header title and header actions.